appinventor/appengine/src/com/google/appinventor/server/DownloadServlet.java

```diff
@@ -96,10 +111,12 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOExc
     CACHE_HEADERS.setNotCacheable(resp);
     resp.setContentType(CONTENT_TYPE);
 
-    RawFile downloadableFile;
+    RawFile downloadableFile = null;
 
     String userId = null;
 
+    int status_code = HttpServletResponse.SC_OK;
```

Comment:
A minor thing, but the `status_code` variable doesn't follow the same naming convention as the rest of the variables.